templates/partials/filters/hideNull.html

```diff
@@ -1,5 +1,5 @@
-<div class="results-info__null">
-  <input id="null-checkbox" type="checkbox" name="sort_hide_null" checked>
+<div class="js-filter filter">
+  <input id="null-checkbox" type="checkbox" name="sort_hide_null" value="true" checked>
```

Comment:
FWIW, if this toggle lives in the filter set, the `checked` attribute won't have any effect, since initial filter values are pulled from the URL on page load.